catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/ImageMetacardTypeTest.java

```diff
@@ -83,6 +83,6 @@ public void testImageAttributes() {
     descriptors.addAll(AbstractNitfMetacardType.getDescriptors(StdidcAttribute.getAttributes()));
     assertThat(
         imageCardType.getAttributeDescriptors(),
-        containsInAnyOrder(descriptors.toArray(new AttributeDescriptor[descriptors.size()])));
+        containsInAnyOrder(descriptors.toArray(new AttributeDescriptor[0])));
```

Comment:
✏️ I usually prefer comments explaining why it's safe when we just put in index 0 like these

Reply:
@ethantmanns this isn't accessing index zero, its just creating a new zerosized array (which the toArray javadoc says is okay and will create a new sized array based off the type of the passed in array).

Reply:
You're right. I read this too fast